tests/units/feature_extraction/test_feature_calculations.py

```diff
@@ -473,6 +473,47 @@ def test_fft_coefficient(self):
         six.assertCountEqual(self, list(res.index), expected_index)
         self.assertIsNaN(res['coeff_10__attr_"real"'])
 
+    def test_fft_aggregated(self):
+        param = [
+            {"aggtype": "centroid"},
+            {"aggtype": "variance"},
+            {"aggtype": "skew"},
+            {"aggtype": "kurtosis"}
+        ]
+        expected_index = ['aggtype_centroid', 'aggtype_variance', 'aggtype_skew', 'aggtype_kurtosis']
```

Comment:
the expected index needs to have quotation marks around the attributes

Reply:
Ok, I will push now, please note the tests will fail because I'm in the middle of fixing the kurtosis and skew calculations (it was incorrect to use the pandas.Series skew/kurtosis functions that I had committed).